api-reference/endpoint/create_event.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: "Create Event"
+api: "POST https://game-server-dev.fly.dev/api/v1/event"
+description: "This endpoint create an Event."
+---
+
+### Parameters
+<ParamField header="Authorization" type="string" required>
+  This is the Authorization token of customer.
+</ParamField>
+
+### Body
+<ParamField body="questId" type="string" required>
+  This is the Id of the Quest
+</ParamField>
+
+<ParamField body="userId" type="string" required>
+  This is the Id of the User
+</ParamField>
+
+<ParamField body="itemId" type="string" required>
+  This is the Id of the Item
+</ParamField>
+
+<ParamField body="points" type="number" required>
+  This is the Number of Point user got
+</ParamField>
+
+### Response
+
+<ResponseField name="eventId" type="string" required>
+ This is Id of the Event.
+</ResponseField>
+
+<ResponseField name="questId" type="string" required>
+ This is Id of the Quest.
+</ResponseField>
+
+<ResponseField name="userId" type="string" required>
+ This is Id of the User.
+</ResponseField>
+
+<ResponseField name="customerId" type="string" required>
+ This is Id of the Customer.
+</ResponseField>
+
+<ResponseField name="status" type="string" required>
+ This is status of event.
+</ResponseField>
+
+<ResponseField name="points" type="number" required>
+ This is the Number of Point user got
```

Comment:
This is the number of points the user has

Reply:
Done